polyglot/piranha/demo/find_replace_custom_cleanup/go/configurations/rules.toml

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2022 Uber Technologies, Inc.
+#
+# <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
```

Comment:
I was wondering, we can do this refactoring as one structural find/replace itself right?
why are we doing two steps here?

Reply:
I'm not sure we can do it in one step as we need to replace two nodes. Consider the AST for `fmt.Print("string\n")`:

```
(call_expression
    function: (selector_expression 
        operand: (identifier)
        field: (field_identifier) @field_id
    )
    arguments: (argument_list
        (interpreted_string_literal 
        (escape_sequence)) @escape_seq
    )
)
```

The two replaces:

1. replaces `@field_id` (`Print`) with `Println` : `fmt.Print("string\n")` -> `fmt.Println("string\n")`
1. replaces `@escape_seq` (`"\n"`) with `""`: `fmt.Println("string\n")` -> `fmt.Println("string")`